remote/conftest.py

```diff
@@ -0,0 +1,133 @@
+# coding=utf-8
+"""Pytest customizations and fixtures for tests to execute on remote hosts."""
+import os
+import pytest
+from camayoc import utils, config
+from time import sleep
+from remote.constants import (
+    DATA_CENTER,
+    CLUSTER,
+    HOST,
+)
+
+
+@pytest.fixture
+def isolated_filesystem():
+    """Fixture that creates a temporary directory.
+
+    Changes the current working directory to the created temporary directory
+    for isolated filesystem tests.
+    """
+    with utils.isolated_filesystem() as path:
+        yield path
+
+
+@pytest.fixture(scope='session', autouse=True)
+def manage_systems():
+    """Fixture that manages remote systems.
+
+    At the beginning of a session, hosts will be turned on. We must wait for
+    the machines to complete booting so there is about 30 seconds of overhead
+    at the beginning before test begin executing.
+
+    At the end of the session, they are turned off.
+
+    Hosts must have a "provider" specified in the config file, otherwise they
+    will be skipped and assumed to be accessible by rho.
+
+    Hosts that are powered on at the beginning of the session are then powered
+    down at the end of the session.
+
+    Example::
+
+        #/home/user/.config/camayoc/config.yaml
+        rho:
+          auths:
+            - username: root
+              name: sonar
+              sshkeyfile: /home/elijah/.ssh/id_sonar_jenkins_rsa
+          hosts:
+            - hostname: string-name-of-vm-on-vcenter
+              ip: XX.XX.XXX.XX
+              provider: vcenter
+              facts:
+                connection.port: 22
+                cpu.count: 1
+          profiles:
+            - name: profile1
+              auths:
+                - auth1
+              hosts:
+                - XX.XX.XXX.XX
+         vcenter:
+          hostname: **Required** -- url to vcenter instance
+          username: **optional** -- will first try environment variable $VCUSER
+          password: **optional** -- will first try environment variable $VCPASS
+
+    *Warning*
```

Comment:
Here you can use the warning directive:

```rst
.. warning::

    First line of the warning
    Second line of the warning
```

Reply:
I tried that an sphinx got mad, it may have been before I installed the developer requirements. I'll try again.